js/ui/grid_core/ui.grid_core.utils.js

```diff
@@ -701,4 +702,32 @@ export default {
 
         return lookupDataSource;
     },
+
+    logColumnsDeprecatedWarningIfNeed(componentName, columnsOptions) {
```

Comment:
We already have the method for logging deprecation warnings
But it is implemented in the ```core/component.js```
[Link to the implementation](https://github.com/DevExpress/DevExtreme/blob/23_1/js/core/component.js#L133)

Let's try to use it instead of adding our custom implementation
Maybe we can prepare options for ```_logDeprecatedOptionWarning``` in utils and call it from the component